outport/nilOutport.go

```diff
@@ -0,0 +1,64 @@
+package outport
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core/statistics"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/outport/drivers"
+	"github.com/ElrondNetwork/elrond-go/outport/types"
+)
+
+type nilOutport struct{}
+
+// NewNilOutport --
+func NewNilOutport() OutportHandler {
+	return new(nilOutport)
+}
+
+// SaveBlock -
```

Comment:
`SaveBlock does nothing`, and for the others (optional).

Reply:
Not optional, mandatory as to comply with the rest of the project :D

Reply:
done.